js/components/eiti-bar-chart.js

```diff
@@ -268,14 +264,6 @@
 
     xAxis.selectAll('path, line')
         .attr('fill', 'none');
-
-    svg.append('g')
-      .attr('class', 'x-axis-label')
-      .append('text')
-        .text(xAxisLabel)
-        .attr('transform', function(d) {
-          return 'translate(' + [labelOffset, fullHeight] + ')';
-        });
```

Comment:
<img width="223" alt="screen shot 2016-09-02 at 4 36 27 pm" src="https://cloud.githubusercontent.com/assets/4803473/18218917/6d2f253c-712b-11e6-989e-b8e1a99e662c.png">

In addition, the text looks like it is very close to the figcaption